rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -241,6 +250,11 @@ rclcpp::remove_ros_arguments(int argc, char const * const argv[])
     alloc.deallocate(nonros_argv, alloc.state);
   }
 
+  ret = rcl_arguments_fini(&parsed_args);
+  if (RCL_RET_OK != ret) {
+    exceptions::throw_from_rcl_error(ret, "failed to cleanup parsed arguments, leaking memory: ");
```

Comment:
Is the `: ` needed in this case? I think this adds it automatically: https://github.com/ros2/rclcpp/blob/45dcd0c6e58bf54d771ae8635d30cc91f8a08e2d/rclcpp/src/rclcpp/exceptions.cpp#L60